tools/local-network/README.md

```diff
@@ -1,14 +1,25 @@
 ## Intro
 
-This directory contains shell scripts for running a local 5-node mesh network and a `mobilecoind` instance.
+This directory contains scripts for running a local mobilecoin consensus network and a `mobilecoind` instance.
+The nodes are run in debug mode, so it is only intended to be used for development and testing purposes.
```

Comment:
The script actually defaults to release mode.

Reply:
Was referring to this warning:
```
2020-04-24 19:32:17.520802775 UTC ERRO Enclave will be started in debug mode, there is no privacy, mc.local_node_id: localhost:3304, mc.app: consensus-service, mc.module: consensus_service, mc.src: consensus/service/src/bin/main.rs:74
```

Reply:
I don't know how many people would think about the enclave mode, versus the build configuration.

Reply:
"SGX in emulation mode"?

Reply:
Yeah - maybe to disambiguate, you could say "The enclaves are built in pre-release mode, which provides debug symbols for debugging purposes."

Reply:
The way to make that warning go away is to build with `IAS_MODE=PROD`.